Dockerfile

```diff
@@ -7,3 +7,5 @@ RUN apk --no-cache --update add bash curl less groff jq python py-pip && \
   mkdir /root/.aws && \
   aws --version && \
   s3cmd --version
+
+CMD /bin/sh -c
```

Comment:
It turns out I was wrong and doing this doesn't work well enough.

Another option which I think does work is `ENTRYPOINT []` and not setting a command. I think this might be subtly different than the previous behavior, but I'm not sure how.

For reference, here's the behavior for each option:

| image | `docker run` | `docker run aws` | `rkt run --interactive` | `rkt run -- aws` |
|---------|-------------------|-----------------------|----|----|
| old (null cmd+ep) | `No command specified` | ✔️ | `app has no executable` | ✔️ |
| cmd /bin/sh -c | `-c requires an argument` | ✔️ | `-c requires an argument` | `-c requires an argument` |
| entrypoint [] | `No command specified` | ✔️ | `app has no executable` | ✔️ |

Notably, running `sudo rkt run --insecure-options=image $image -- aws --version` with this change produces the `-c requires an argument` error, and previously that worked.

I tested this with the follwoing script if you're curious (and in case I messed up):

<Details>

```sh
unset="quay.io/euank/test:both-unset" # mirror of quay.io/coreos/awscli:latest
empty="quay.io/euank/test:entrypoint-empty" # ENTRYPOINT []
shc="quay.io/euank/test:sh-c" # Build+push of PR 17

run() {
	name=$1
	image=$2

	echo "Running $name"
	echo -e "empty\tdocker\t$(docker run $image 2>&1)"
	echo -e "empty\trkt   \t$(rkt run --insecure-options=image $image 2>&1)"
	echo -e ""
	echo -e "aws\tdocker\t$(docker run -it $image aws --version 2>&1)"
	echo -e "aws\trkt   \t$(rkt run --interactive --insecure-options=image $image -- aws --version 2>&1)"
	echo -e "\n======\n\n"
}

run "null cmd+entrypoint" "$unset"
run "[] entrypoint      " "$empty"
run "sh -c cmd          " "$shc"
```

Reply:
Thanks for doing the due diligence on this @euank, i didn't have rkt readily available here.

Reply:
Nice work @euank. Thank you.